app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/ActivityTrackingController.java

```diff
@@ -129,6 +131,34 @@ private int deleteFieldsLT(String path, String field) {
         });
     }
 
+    private void writeBatch(TreeMap<String, Object> batch) {
+        dbi.inTransaction((conn, status) -> {
+            PreparedBatch insert = conn.prepareBatch("INSERT into jsondb (path, value, ovalue) values (:path, :value, :ovalue)");
+            for (Map.Entry<String, Object> entry : batch.entrySet()) {
+                String key = "/activity" + entry.getKey() + "/";
```

Comment:
not sure but shouldn't this be `"/activity/" + entry.getKey() + "/"`;

Reply:
Yeah it's right.  entry.getKey() = "/somethign"